src/main/java/io/nats/client/impl/NatsConnection.java

```diff
@@ -1239,31 +1240,31 @@ Message requestInternal(String subject, Headers headers, byte[] data, Duration t
      */
     @Override
     public CompletableFuture<Message> request(String subject, byte[] body) {
-        return requestFutureInternal(subject, null, body, null, cancelAction, true);
+        return requestFutureInternal(subject, null, body, null, cancelAction, true, true);

```

Comment:
Should this and these below also be adapted to use `options.forceFlushOnRequest()`?